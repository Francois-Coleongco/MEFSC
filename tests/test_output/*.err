DIDNT BAIL WE HAVE VALID KEYSSS YAYAYYYYY

this was username length 5
this was password length 7
datalength sent 22


datalength sent 24


data length of notif is: 20datalength sent 20


datalength sent 20


made it here in WTFS_Handler
couldn't open the file
error in encrypt_and_send_to_server
data length of notif is: 20datalength sent 20


