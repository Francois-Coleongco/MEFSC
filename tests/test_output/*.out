

enter username:
enter password:
enter your intention (1 == read || 2 == write)
made salt, creating key
enter file name to send to server:
made salt, creating key
perform another action?
